### PR TITLE
Add 1 degree

### LINKED ISSUE #49: Adding regression tests using 1-degree MOM6 and CICE5 components
## Description
The 1-degree MOM6 and CICE5 components will be used in DATM-MOM6-CICE5.

### PR DESCRIPTION
**Pull request by binli2337**
## Description
One-degree regression tests are added to the regression test system. 

Fixes #49  

## Testing:
New baselines are generated on Hera and Orion.

The following 4 new regression tests are added:
cpld_datm_mom6_cice5_100_cold_cfsr
cpld_datm_mom6_cice5_100_2d_cfsr  
cpld_datm_mom6_cice5_100_cold_gefs
cpld_datm_mom6_cice5_100_2d_gefs  

Co-authored by: jiandewang

### CODE REVIEW

File: parm/MOM_input_template100 [hunk 1]
```diff
@@ -0,0 +1,888 @@
+/* This input file provides the adjustable run-time parameters for version 6 of
+   the Modular Ocean Model (MOM6), a numerical ocean model developed at NOAA-GFDL.
+   Where appropriate, parameters use usually given in MKS units.
+
+   This particular file is for the example in OM4_025.
```
Comment (junwang-noaa) [inline parm/MOM_input_template100]: Is this file for quarter degree? If the setting is changed for 1 degree, maybe add a comment line to specify what has been changed from 025 (grid info, time step, phys scheme, etc),

**Review by junwang-noaa — APPROVED** [review]
Please see the question,  otherwise the code looks good to me.

**Comment by DeniseWorthen** [comment]
The MOM_input file for 1deg appears to be a copy of the 1/4 deg input but w/ the domain size and data files changed. Jiande had explained that each MOM resolution required it's own MOM_input. I believe we should obtain a 1-deg MOM_input from Jiande.

**Comment by DeniseWorthen** [comment]
There seem to be multiple differences in things like the value of mixing parameters etc between this file and what Jiande gave me for the 1deg MOM6. I don't know enough about all of MOM's parameter settings to know which can be safely left at the 025 values and which need to be changed (and to what) for the lower resolution.

**Comment by binli2337** [comment]
New baselines are generated with the new MOM_input_template100 file.

This pull request is ready for review.

**Review by DeniseWorthen — APPROVED** [review]

**Pull request merged**